taker-frontend/src/components/Trade.tsx

```diff
@@ -68,19 +65,6 @@ interface TradeProps {
     isLongSubmitting: boolean;
 }
 
-interface AlertBoxProps {
-    title: string;
-    description: string;
-}
-
-function AlertBox({ title, description }: AlertBoxProps) {
-    return (<Alert status="error">
-        <AlertIcon />
-        <AlertTitle mr={2}>{title}</AlertTitle>
-        <AlertDescription>{description}</AlertDescription>
-    </Alert>);
-}
```

Comment:
Extracting something like this could easily go into a separate commit. Makes review easier if refactorings are separate from features.